kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/mwp/MwpClient.java

```diff
@@ -41,7 +32,13 @@
   private Logger logger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "logger" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)